internal/nodes/dequeue_ingestion.go

```diff
@@ -160,12 +160,29 @@ func DequeueIngestion(key string, fullsync bool) {
 
 	// L4Rule CRD processing.
 	if objType == lib.L4Rule {
+
+		// L4 Service
```

Comment:
@swathinsankaran: should we return from here in case of `found` is false? As there is no L4Rule associated, so no point in processing ahead.

Reply:
I have addressed the comment.